addons/pos_restaurant/static/src/app/control_buttons/control_buttons.js

```diff
@@ -58,18 +58,18 @@ patch(ControlButtons.prototype, {
         this.currentOrder.takeaway = isTakeAway;
         this.currentOrder.update({ fiscal_position_id: isTakeAway ? takeawayFp : defaultFp });
     },
-    editOrderNote(order) {
+    editFloatingOrderName(order) {
         this.dialog.add(TextInputPopup, {
-            title: _t("Edit order note"),
-            placeholder: _t("18:45 John 4P"),
-            startingValue: order.note || "",
+            title: _t("Edit Order Name"),
+            placeholder: _t("Emma's Birthday Party"),
```

Comment:
I don't think you have to change the placeholder. It's not mentioned in the spec. Unless I'm wrong.